core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html

```diff
@@ -126,6 +126,7 @@ <h2 class="flex align-items-center gap-1 uppercase">
                     [required]="true"
                     [text]="variant.name"
                     (textChanged)="editVariantName($event, variant, vm.experimentId)"
+                    hideErrorMsg="true"
```

Comment:
Test?

Reply:
Shouldn't it be?

```suggestion
                    [hideErrorMsg]="true"
```

Reply:
- Added test.
-  Changed input name and logic, removed.